demo/list/full-view.html

```diff
@@ -72,41 +78,36 @@
     properties: {
 
       sharedElements: {
-        type: Object,
-        value: function() {
-          return {
-            'hero': this.$.main
-          };
-        }
+        type: Object
       },
 
       animationConfig: {
-        type: Object,
-        value: function() {
-          return {
-            'entry': [{
-              name: 'fade-in-animation',
-              node: this.$.button
-            }, {
-              name: 'hero-animation',
-              id: 'hero',
-              toPage: this
-            }],
-
-            'exit': [{
-              name: 'fade-out-animation',
-              node: this.$.button
-            }, {
-              name: 'scale-down-animation',
-              node: this.$.main,
-              transformOrigin: '50% 50%',
-              axis: 'y'
-            }]
-
-          }
-        }
+        type: Object
       }
+    },
 
+    attached: function() {
+      this.sharedElements = { 'hero': this.$.main };
```

Comment:
this is different from 1.x, as at every `attached` you'd override `this.animationConfig` and `this.sharedElements`.
WDYT of just having a check before this line like `if (this.animationConfig) return;`?